projects/storefrontapp-e2e-cypress/cypress/helpers/product-search-rating-flow.ts

```diff
@@ -1,20 +1,24 @@
+import { clickSearchIcon } from './product-search';
 import { PRODUCT_LISTING } from './data-configuration';
 
 export const resultsTitle = 'cx-breadcrumb h1';
-export const tabsHeaderList = 'cx-product-tabs .details > h3';
+export const tabsHeaderList = 'cx-tab-paragraph-container > h3';
+
+const productSelector = 'cx-product-list-item';
 
 export function productRatingFlow(mobile?: string) {
-  // Search for a product
-  cy.get('cx-searchbox input').type('DSC-N1{enter}');
+  clickSearchIcon();
+  const productName = 'DSC-N1';
```

Comment:
I guess this is preference, and maybe following the convention we have right now, but maybe put that above (outside the function)

Reply:
No, because no other function is using this.